test/navigation/ExpensesNavigatorSpec.scala

```diff
@@ -262,53 +216,33 @@ class ExpensesNavigatorSpec extends SpecBase {
               navigator.nextPage(StaffCostsDisallowableAmountPage, mode, emptyUserAnswers, taxYear, businessId) shouldBe expectedResult
             }
           }
-          "the page is StaffCostsCYAPage" - {
-            "navigate to the SectionCompletedStateController" in {
-              val expectedResult = SectionCompletedStateController.onPageLoad(
-                taxYear,
-                businessId,
-                ExpensesStaffCosts.toString,
-                mode
-              )
-
-              navigator.nextPage(StaffCostsCYAPage, mode, emptyUserAnswers, taxYear, businessId) shouldBe expectedResult
-            }
-          }
         }
 
         "DisallowableSubcontractorCosts journey" - {
           "the page is ConstructionIndustryAmountPage" - {
-            "some expenses were claimed to be disallowable and accounting type is Accrual" - {
+            "some expenses were claimed to be disallowable" - {
               "navigate to the ConstructionIndustryDisallowableAmountPage" in {
                 val data        = Json.obj(businessId.value -> Json.obj("disallowableSubcontractorCosts" -> "yes"))
                 val userAnswers = UserAnswers(userAnswersId, data)
 
                 val expectedResult = construction.routes.ConstructionIndustryDisallowableAmountController.onPageLoad(taxYear, businessId, mode)
 
-                navigator.nextPage(ConstructionIndustryAmountPage, mode, userAnswers, taxYear, businessId, Some(Accrual)) mustBe expectedResult
+                navigator.nextPage(ConstructionIndustryAmountPage, mode, userAnswers, taxYear, businessId) mustBe expectedResult
               }
             }
             "navigate to the ConstructionIndustryCYAPage" - {
-              "if all expenses were claimed as allowable" in {
+              "all expenses were claimed as allowable" ignore {
```

Comment:
How come we have marked this test (and the one below it) as ignored?

Reply:
Thanks, these are bad stash/merges